src/components/Legend.js

```diff
@@ -22,9 +22,23 @@ export default class Legend {
       .attr('x2', '100%')
       .attr('y2', '0%');
 
-    legendGradient.append('stop')
-    .attr('offset', '0%')
-    .attr('style', `stop-color:${this.map.options.countries.attr.fill.min};stop-opacity:1`);
+    if (this.map.options.countries.attr.fill.minNegative &&
+        this.map.options.countries.attr.fill.maxNegative) {
+      legendGradient.append('stop')
+          .attr('offset', '0%')
+          .attr('style', `stop-color:${this.map.options.countries.attr.fill.maxNegative};stop-opacity:1`);
+
+      legendGradient.append('stop')
+          .attr('offset', '49%')
```

Comment:
Putting it as half is a wrong assumption if we have a minimum -20 and a maximum of 60. (for this use case the cut between the negative and positive is 25%). That's why the legend broke, because it's a scale only from 0 -> maxValue.